### DISCUSSION TITLE
【分享】分享的網址有問題

### DISCUSSION THREAD

**Discussion by zengjixiang** [tangly1024/NotionNext#1997] [Q&A]
**描述bug**
【此项必填】简单说明bug的现象、相关的错误提示、日志等
分享文章時會自動將網域導到原博主的網站，也沒有看到相關的設定可以調整。
而且用手機瀏覽，有時會無法顯示文章列表，必須再更新一次網頁才行。

**复现步骤**
【此项必填】出现这个bug的操作步骤
點選文章的分享圖示時

**期望的正常结果**
【此项必填】希望按这个步骤，正常操作结果是什么
分享到指定的社群

**截图**
【可选】相关的页面，应该的结果

**环境**

- 【必填】NotionNext版本 [例如. 4.0.18]
- 4.1.4
- 【必填】主题 [例如. hexo]
- 預設
- 【可选】操作系统: [例如. iOS, Android, macOS, windows]
- 【可选】浏览器 [例如. chrome, safari, firefox]

**补充说明**
【可选】与问题相关的其它说明

**Comment by tangly1024** [comment]
> 分享文章時會自動將網域導到原博主的網站

这个需要在blog.config.js 中[配置您的站点地址](https://docs.tangly1024.com/article/notion-next-guide#1b292a69b6094dda98a9d8eb7db7880d)

https://github.com/tangly1024/NotionNext/blob/c22084e8d48d3520bc516d4015828281fdabf5a9/blog.config.js#L22


> 手機瀏覽，有時會無法顯示文章列表，必須再更新一次網頁才行。

请问使用的是哪一个主题，我怀疑是动画卡住了

**Comment by zengjixiang** [comment]
> > 分享文章時會自動將網域導到原博主的網站
> 
> 这个需要在blog.config.js 中[配置您的站点地址](https://docs.tangly1024.com/article/notion-next-guide#1b292a69b6094dda98a9d8eb7db7880d)
> 
> https://github.com/tangly1024/NotionNext/blob/c22084e8d48d3520bc516d4015828281fdabf5a9/blog.config.js#L22
> 
> > 手機瀏覽，有時會無法顯示文章列表，必須再更新一次網頁才行。
> 
> 请问使用的是哪一个主题，我怀疑是动画卡住了

對，我猜應該是這個原因，因為不是每次無法正常顯示。
（補充說明，主題是用 Next)
